### ISSUE TITLE
Aspect ratio "none"

### ISSUE THREAD

**Issue by adamrobinson** [webdna/tailwindcss-aspect-ratio#10]
Hi.

Would be nice to be able to remove an aspect ratio using the same class name syntax e.g. `aspect-ratio-none`.  Ive not been able to make my own config that outputs `padding-top: 0%` to do this.

I know i could do this with `aspect-ratio-16/9 lg:pt-0` but you need to know what the aspect ratio class is doing in order to know that `pt-0` is negating it.  `aspect-ratio-16/9 lg:aspect-ratio-none` is easier to understand at a glance. 

Thanks

**Comment by adamrobinson** [comment]
Sorry meant `lg:pb-0` to remove the padding.

**Comment by benface** [comment]
Hey @adamrobinson, thanks for the suggestion! I've added this in the [`next` branch](https://github.com/webdna/tailwindcss-aspect-ratio/tree/next) which will be released as v3.0 shortly after Tailwind 1.2. In the meantime, you could add it as a custom utility in your CSS:

```css
@responsive {
  .aspect-ratio-none {
    @apply pb-0;
  }
}
```
